page/about-jquery/using-jquery.md

```diff
@@ -0,0 +1,155 @@
+
+  // Your code here
+
+  event.preventDefault();
+
+});
+```
+
+```
+// main.css
+a.active {
+  font-weight: bold;
+}
+```
+
+**Important:** *You must place the remaining jQuery examples inside the `ready` event so that your code executes when the document is ready to be worked on, as explained in the [previous setup tutorial](http://learn.jquery.com/about-jquery/how-to-setup-jQuery/).*
+
+Add the [.addClass()](http://api.jquery.com/addClass/) call inside the `ready` event:
+
+```
+$( "a" ).addClass( "active" );
+```
+
+All `<a>` elements are now bold. If you right click on a link and select "Inspect Element" you'll see that the class of "active" has been added to all `<a>` elements.
+
+To remove an existing class, use [.removeClass()](http://api.jquery.com/removeClass/):
+
+```
+$( "a" ).removeClass( "active" );
+```
+
+Let's bring this all together now. What if we only want the most recently clicked link to turn bold? jQuery can alter groups of elements by selecting all of them, but it can also point to a specific element using the `$(this)` referant.
+
+```
+var $links = $( "a" ).on( "click", function makeActive( event ) {
+
+  $links.removeClass( "active" );
+  $( this ).addClass( "active" );
+
+  event.preventDefault();
+
+});
+```
+
+First, we save the selection of all `<a>` elements to the variable `$links`, so we can reuse the selection later without having to query the DOM again.
+
+Next, we remove the "active" class from all `<a>` elements, guaranteeing that the link clicked previously is no longer bold or "active".
+
+`$(this)` then refers to the specific link that was clicked on, and so applies the HTML class only to that link, resulting in the desired effect of the "active" class moving from link to link as the user changes the focus.
+
+This strategy of altering an HTML class is used all the time to alter an element based on user actions and events, like hiding content until a user clicks to expand it, or showing a different menu to a user based on whether or not they're signed in.
+
+### Special Effects
+
+jQuery also provides some handy visual [effects](http://api.jquery.com/category/effects/) to help you make your websites stand out. For example, if you create a click handler of:
+
+```
+$( "a" ).on("click", function fadeAway( event ) {
+
+  event.preventDefault();
+
+  $( this ).hide( "slow" );
+
+});
+```
+
+The link slowly disappears when clicked.
+
+## Callbacks and Functions
+
+Unlike many other programming languages, JavaScript enables you to freely pass functions around to be executed at a later time. A *callback* is a function that is passed as an argument to another function and is executed after its parent function has completed. Callbacks are special because they patiently wait to execute until their parent finishes. Meanwhile, the browser can be executing other functions or doing all sorts of other work.
+
+To use callbacks, it is important to know how to pass them into their parent function.
```

Comment:
I feel like we could link to a good article either on our own site or on MDN for this topic also